src/test/test-argv-util.c

```diff
@@ -113,6 +113,19 @@ TEST(rename_process) {
         test_rename_process_one("foo", 1); /* should always fit */
         test_rename_process_one("this is a really really long process name, followed by some more words", 0); /* unlikely to fit */
         test_rename_process_one("1234567", 1); /* should always fit */
+
+}
+
+
+/* Test for the function argv_looks_like_help */
+static void test_argv_looks_like_help(const char *args, int argc) {
+        char *k[] = {(char*)args};
```

Comment:
just merge this below, no need for another function

Reply:
Do you mean I should get rid of the TEST(argv_help) function, and instead call test_argv_looks_like_help from the TEST(rename_process) function?
I just realized that PRs on the project have to go through lots of checks, so I want to make sure I'm doing the right thing. And more importantly, I would rather save your precious time sir than make assumptions.

Reply:
other way around, just do the checks inside `TEST(argv_help)`

Reply:
I'm thinking you're saying I should have something like:
TEST(argv_help){
assert_se(argv_looks_like_help({"prog"},  1) == true);
}

Then get rid of the static void function I earlier created.

Is that right sir?

Reply:
something like that yes

Reply:
Thank you so much sir. I'll get on it.
I need to figure out how to pass an array of strings directly.
I'll be through with that soon.

Thank you sir.

Reply:
I wrote a new test sir:

Reply:
![new_test](https://user-images.githubusercontent.com/109356020/224505230-7a3678b3-8441-4ecd-bc1d-c4e9c6379c83.PNG)

Reply:
Should I push sir?

Reply:
there's no need to ask, just push when you have changes

Reply:
Alright sir. It's being a great learning experience.